README.md

```diff
@@ -1,2 +1,31 @@
-# advisory-group
-Enterprise Advisory Group
+# Enterprise Advisory Group
+
+The Enterprise Advisory Group is a group of corporate representatives in roles of leadership responsible managing for large-scale deployments of Node.js in an Enterprise environment. As a group, we strive to provide timely feedback to the [TSC](https://github.com/nodejs/TSC), the [CommComm](https://github.com/nodejs/community-committee), the [Node.js project](https://github.com/nodejs/node) and the Node.js Foundation about the unique needs surrounding running Node.js in an Enterprise environment.
+
+This group is a part of the [Node.js End User Feedback Inititative](https://github.com/nodejs/user-feedback) which is under the charter of the [Node.js Community Committee (CommComm)](https://github.com/nodejs/community-committee). Thank you for your interest and support. Please [create an issue](https://github.com/nodejs/user-feedback/issues) in this repo if you are interested in dedicating 5+ hours a month to during the formation process.
+
+### Governance and Contributing
+The Enterprise Advisory Group has adopted the core governance and contributing policies of Node.js. This group is a part of the [Node.js End User Feedback Inititative](https://github.com/nodejs/user-feedback) which is under the charter of the [Node.js Community Committee (CommComm)](https://github.com/nodejs/community-committee).
+
+Members are expected to be able to provide feedback on proposed changes in the Node.js project and gather input on how those changes will impact teams, business goals and application performance at scale and in the Enterprise context. Members are expected to attend regular advisory meetings to provide feedback and respond to occational outreach requests. 
+
+### Enterprise Advisory Group Team Members
+
+* [dshaw](https://github.com/dshaw) - **Dan Shaw** &lt;dshaw@dshaw.com&gt; - Node.js End User Feedback coordinator - Node.js at Large ✨
+* [jasnell](https://github.com/jasnell) - **James Snell** - TSC - nearForm
+* **Scott Anderson** - MasterCard
+* **Chris Castle** - Heroku
+* **Monica Ene-Pietrosanu** - Intel
+* **Scott Hammond** - Joyent
+* **Trevor Livingston** - Expedia Group / HomeAway
+* **Todd Moore** - IBM
+* **Ahmad Nassri** - TELUS
+* **Sarah Novotny** - Google
+* **Gaurav Seth** - Microsoft
+* **Laurie Voss** - npm
+
+## Additional Context
+
+The organization and formation of this initiative was kicked off by James Snell ([@jasnell](https://github.com/jasnell)) and Dan Shaw ([@dshaw](https://github.com/dshaw)). We are greatful for their leadership.
```

Comment:
`greatful` -> `grateful`

Reply:
also who is `We` in this context? considering that you, @dshaw, are proposing this, it feels... weird, to say the least

Reply:
Whatever the intent, appearing to express gratitude for your own leadership is something to avoid. I don't see any reason not to eliminate the entire "Additional Context" section.

Reply:
Removed